wandb/wandb_torch.py

```diff
@@ -354,12 +355,19 @@ def after_forward_hook(module, input, output):
                 elif isinstance(output[0], list) and hasattr(output[0][0], "grad_fn"):
                     graph.criterion = output[0][0].grad_fn
 
+            # log graph and remove hook
+            hook = self._graph_hooks.pop(id(module), None)
+            if hook is not None:
+                hook.remove()
+
+            if not self._graph_hooks:
+                # we went through the entire graph
+                wandb.run.summary["graph_%i" % graph_idx] = self
```

Comment:
We used to have `if wandb.run` are there cases where we don't have a run yet?

Reply:
I don't think so because of [this line](https://github.com/wandb/client/blob/39c658e66fd38716eae52782968aafbe0ae7c6a0/wandb/sdk/wandb_watch.py#L45)

Reply:
… unless a run was finished, in which case we may be ok to have an error returned?

Reply:
This is probably fine.